nakadi/event_stream.py

```diff
@@ -7,72 +7,94 @@
 import sys
 
 
-def create_stream_generator(kafka_pool, topic, partition, start_from, opts):
+def create_stream_generator(kafka_pool, topic, cursors, opts):
+
+    topics = {}
+    for cursor in cursors:
+        topic_partition = (topic, int(cursor['partition']))
+        topics[topic_partition] = int(cursor['offset'])
```

Comment:
Another question, do we want to scramble the offsets so that they do not look like numbers?

Just base64 encoding of the numeric value of the offset would be sufficient and easy enough.